### ISSUE TITLE
Error Message Parameter Possibly Incorrect 

### ISSUE THREAD

**Issue by kpm1985** [astralway/phrasecount#36]
https://github.com/astralway/phrasecount/blob/7e301ab9408ee04dfbb1396c10b0f054f38cedcf/src/main/java/phrasecount/cmd/Load.java#L32

I'm just eyeballing here, but it looks like the line above should reference args[1] instead of the args[0].

**Comment by kpm1985** [comment]
https://github.com/astralway/phrasecount/blob/7e301ab9408ee04dfbb1396c10b0f054f38cedcf/src/main/java/phrasecount/cmd/Load.java#L23-L33

Noticing line 32 where it references args[0] and compare with line 23 shows the conflict. I think its just an index off by one thing, change line 32 to args[1]. I'd welcome a sanity check, I've had the California flu :(

**Comment by keith-turner** [comment]
>  I'd welcome a sanity check, I've had the California flu :(

@kpm1985 I agree that error message does look incorrect.  Hope you are feeling better.
